#goal definition
geometry_msgs/PoseStamped target_pose
---
#result definition
---
#feedback
geometry_msgs/PoseStamped base_position